type.proto

```diff
@@ -65,14 +66,15 @@ message Prediction {
 }
 
 message Object {
+  reserved 7;
   int32 id = 1;
   LabelType label = 2;
   float confidence = 3;
   BoundingBox bbox = 4;
   Vector3 velocity = 5;
   TrackingStatus tracking_status = 6;
-  google.protobuf.Timestamp timestamp = 7;
   float yaw_rate = 8;
+  google.protobuf.Duration time_offset = 9;
```

Comment:
Can we keep the index of `time_offset` to 7?
I believe `google.protobuf.Timestamp timestamp = 7;` was not officially released, doesn't it?

Reply:
There are vin associator releases with the field number so SENSR/VINA would be incompatible with the version. Can we reserve the number for a while and then remove it later?

Reply:
It's totally fine to use the new index.
The only down part is that we can not use the reserved number anymore.
If you have a case which  VINA and SENSR version are not synced, then I'm fine to the new index.
But, please make sure one thing.
You have to implement backward compatibility in VINA. Because I couldn't find any place to support backward compatibility in PR2026 in argos.

Reply:
Do you mean https://github.com/seoulrobotics/argos/pull/2029?